mmv1/third_party/terraform/.teamcity/components/builds/build_steps.kt

```diff
@@ -226,3 +226,54 @@ fun BuildSteps.archiveArtifactsIfOverLimit() {
         // https://youtrack.jetbrains.com/issue/KT-2425/Provide-a-way-for-escaping-the-dollar-sign-symbol-in-multiline-strings-and-string-templates
     })
 }
+
+fun BuildSteps.runDiffTests() {
+    if (UseTeamCityGoTest) {
+        step(ScriptBuildStep {
+            name = "Run Tests"
```

Comment:
currently running this on our teamcity test project [here](https://hashicorp.teamcity.com/buildConfiguration/Development_TerraformProviderGoogleDiffTestProject_GOOGLE_BETA_DIFF_TEST_NIGHTLYTESTS_GOOGLEBETA_PACKAGE_ACCESSAPPROVAL/438227)

it seems like we shouuld hvae the build test step be unique for Diff Tests. it could just be as simple as having it be named `Run Diff Tests` instead of leaving it as `Run Tests`